src/cli-sdk/src/commands/query.ts

```diff
@@ -52,6 +67,29 @@ type QueryResult = JSONOutputGraph &
   MermaidOutputGraph &
   HumanReadableOutputGraph & { queryString: string }
 
+const validateExpectedResult = (
```

Comment:
This validation function belongs in `src/cli-sdk/src/config/definitions.ts`, so that incorrect configs will be validated in the same way as all other configurations.

Reply:
Oh, no, this is a different thing, nevermind.